exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -17,9 +17,90 @@
 
 package com.exonum.client;
 
+import com.exonum.binding.common.hash.HashCode;
+import com.exonum.binding.common.message.TransactionMessage;
+import java.net.MalformedURLException;
+import java.net.URL;
+import okhttp3.OkHttpClient;
+
 /**
- * Exonum client main class to start with.
- */
+ * Main class to start using Exonum Light client.
+ * Provides a convenient way for interaction with Exonum framework APIs.
+ * It uses {@linkplain OkHttpClient} internally for REST API communication with Exonum node.
```

Comment:
I'd move such information to the implementation class (we can easily imagine a grpc client, can't we :smiley_cat:).

Reply:
agree :)